rcode-reader/src/main/java/eu/rssw/pct/elements/IndexComponentElement.java

```diff
@@ -4,14 +4,23 @@
 import java.nio.ByteOrder;
 
 public class IndexComponentElement extends AbstractElement {
-  protected int ascending;
-  protected int flags;
-  protected int position;
-
-  protected IndexComponentElement(byte[] segment, int currentPos, int textAreaOffset, ByteOrder order) {
-    this.ascending = segment[currentPos];
-    this.flags = segment[currentPos + 1];
-    this.position = ByteBuffer.wrap(segment, currentPos + 2, Short.BYTES).order(order).getShort();
+  private final int ascending;
+  private final int flags;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "flags" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1068)